package-lock.json

```diff
@@ -2180,17 +2180,17 @@
 			}
 		},
 		"@sebgroup/bootstrap": {
-			"version": "4.0.0",
-			"resolved": "https://registry.npmjs.org/@sebgroup/bootstrap/-/bootstrap-4.0.0.tgz",
-			"integrity": "sha512-iKXixIQL0SVkpXXYnfKxRD+MfYCiLi7kL17mBU9H4Hfw6635WHJjVBmKcHFjoHq2R1sXc4TgqzDtNMurpgoJ2Q==",
+			"version": "5.1.0",
```

Comment:
You can’t use SEB servers in the lock. Just run `npm run package-lock-sanitizer` and commit

Reply:
Sorry I tagged the line of code from mobile which messed up, but if you see the new URL you will find them point at **sebank.se**